packages/wonder-blocks-tabs/src/components/__tests__/tab.test.tsx

```diff
@@ -176,5 +197,39 @@ describe("Tab", () => {
                 expect(tab).not.toHaveAttribute("aria-selected");
             });
         });
+
+        describe("Focus", () => {
+            it("should be focusable if it is selected", async () => {
+                // Arrange
+                render(
+                    <Tab {...props} selected={true}>
+                        Tab
+                    </Tab>,
+                );
+                const tab = await screen.findByRole("tab");
+
+                // Act
+                await userEvent.keyboard("{tab}");
+
+                // Assert
+                expect(tab).toHaveFocus();
+            });
+
+            it("should not be focusable if it is not selected", async () => {
+                // Arrange
+                render(
+                    <Tab {...props} selected={false}>
+                        Tab
+                    </Tab>,
+                );
+                const tab = await screen.findByRole("tab");
+
+                // Act
+                await userEvent.keyboard("{tab}");
```

Comment:
I'm realizing the word `tab` can mean so many different things! 😅

Should we standardize on `userEvent.tab()` vs. `userEvent.keyboard("{tab}")` or `userEvent.keyboard("{Tab}")`?

Reply:
Good call! Went with using `userEvent.tab()`!